spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/GravitinoIcebergCatalog.java

```diff
@@ -157,6 +159,22 @@ public org.apache.spark.sql.connector.catalog.Table loadTable(Identifier ident,
     }
   }
 
+  @Override
+  protected String getDatabase(Identifier sparkIdentifier) {
```

Comment:
is it possible Identifier contains `catalog` info?

Reply:
> is it possible Identifier contains `catalog` info?

it is Spark `Identifier`, and the `Identifier` itself does not directly contain `catalog` info.

Reply:
ok